nodebuilder/daser/config.go

```diff
@@ -0,0 +1,76 @@
+package daser
```

Comment:
Relic from the past. The pkg is called `daser` while the pkgs defining Daser is called `das`. Let's make them consistent as with any other modules we have.